src/extension.ts

```diff
@@ -221,7 +221,7 @@ function initAsyncCommand(context: vscode.ExtensionContext, commandId: string, c
                     output.show();
                     vscode.window.showErrorMessage('An error has occured. Check output window for more details.');
                 } else {
-                    vscode.window.showErrorMessage(errorData.message);
+                    vscode.window.showErrorMessage(`Error: ${errorData.message}`);
```

Comment:
I think you meant to do this on line 219 instead of here?

Reply:
You are correct. 😞 Fixing.